aidial_adapter_bedrock/bedrock.py

```diff
@@ -37,35 +39,50 @@ def _create_invoke_params(self, model: str, body: dict) -> dict:
         }
 
     async def ainvoke_non_streaming(self, model: str, args: dict) -> dict:
+
+        if log.isEnabledFor(DEBUG):
+            log.debug(
+                f"request: {json_dumps_short({'model': model, 'args': args})}"
+            )
+
         params = self._create_invoke_params(model, args)
         response = await make_async(lambda: self.client.invoke_model(**params))
 
-        log.debug(f"response: {response}")
+        if log.isEnabledFor(DEBUG):
+            log.debug(f"response: {json_dumps_short(response)}")
 
         body: StreamingBody = response["body"]
         body_dict = json.loads(await make_async(lambda: body.read()))
 
-        log.debug(f"response['body']: {json.dumps(body_dict)}")
+        if log.isEnabledFor(DEBUG):
```

Comment:
It's not useless **performance-wise**, because it prevents `log.debug` arguments from being computed.

It doesn't matter when you have a literal string with a couple of atomic arguments like this:

```python
log.debug("Hello %s!", user_name)
```

but when we are dealing with logging of large data structures, it's worth to make the shortcut:

```
if log.isEnabledFor(DEBUG):
    log.debug(f"request: {long_expensive_computation(request)}")
```

Reply:
Hmm, I have never considered our inputs to be as huge

But yeah, this could potentially add some value on huge images,  so you're right